modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,219 @@
+status: {}
+#...
+----
+
+. Configure the NUMA Resources Operator, you must create a single `NUMAResourcesOperator` CR on the cluster. The `nodeGroups` configuration in this CR tells the Operator which node pools to manage.
++
+[NOTE]
+====
+Before configuring `nodeGroups`, ensure the specified node pool meets all prerequisites detailed in Section 12.5, "Configuring a single NUMA node policy." The NUMA Resources Operator requires all nodes within a group to be identical. Non-compliant nodes prevent the NUMA Resources Operator from performing the expected topology-aware scheduling for the entire pool.
+
+You can specify multiple non-overlapping node sets for the NUMA Resources Operator to manage. Each of these sets should correspond to a different machine config pool (MCP). The NUMA Resources Operator then manages the schedulable control plane nodes within these specified node groups.
+====
+
+.. For a compact cluster, the compact cluster's master nodes are also the schedulable nodes, so specify only the master pool. Create the following `nodeGroups` configuration in the `NUMAResourcesOperator` CR:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+    - poolName: master
+----
++
+[NOTE]
+====
+Configuring a compact cluster with a worker pool in addition to the `master` pool should be avoided. While this setup does not break the cluster or affect operator functionality, it can lead to redundant or duplicate pods and create unnecessary noise in the system. The worker pool is essentially a pointless, empty MCP in this context and serves no purpose.
+====
+
+.. For an MNO cluster where both control plane and worker nodes are schedulable, you have the option to configure the NUMA Resources Operator to manage multiple `nodeGroups`. You can specify which nodes to include by adding their corresponding MCPs to the `nodeGroups` list in the `NUMAResourcesOperator` CR. The configuration depends entirely on your specific requirements. For example, to manage both the `master` and `worker-cnf` pools, create the following `nodeGroups` configuration in the NUMAResourcesOperator CR:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+    - poolName: master
+    - poolName: worker-cnf 
+----
++
+[NOTE]
+====
+You can customize this list including any combination of `nodeGroups` you want to manage with Topology-Aware Scheduling. To prevent issues with duplicate, pending pods, you must ensure that each `poolName` in your configuration corresponds to an MachineConfigPool (MCP) with a unique node selector label. This label should be applied only to the nodes within that specific pool and not overlap with labels on any other nodes in the cluster. The `worker-cnf` MCP  designates a set of nodes that run telecommunications workloads
```

Comment:
> To prevent issues with duplicate, pending pods,

Do you want to prevent _issues with_ duplicate pods or do you just want to prevent duplicates of pods? Reading it, my suspicion is that the duplication is the issue but that's just my guess.
If I'm right, then should this be
"To prevent duplicate pending pods.... "0